src/Views/gagepage.html

```diff
@@ -240,20 +240,21 @@ <h4 class="modal-title">StreamStats Gage Page</h4>
                         <div class="well well-sm" style="background-color:#EBF0F5;" ng-if="vm.heatChartConfig">
                             <highchart id="chart2" config="vm.heatChartConfig"></highchart>
                         </div>
-                    </center> -->
+                    </center>
                     <center>
-                        <div ng-if="!vm.dischargeChartConfig" class="panel panel-default reportSpinner">
+                        <div ng-if="!vm.chartConfig" class="panel panel-default reportSpinner">
```

Comment:
Should `chartConfig` have been renamed from `dischargeChartConfig`?

Reply:
I was testing different variables names and such and thought I pushed up my changes with them corrected. When I open on my branch now, it shows the right names, but I must not have pushed up my most recent changes..?

Reply:
After coming back to this, I am a bit confused. I believe it should indeed be dischargeChartConfig as this part of the html is dealing with my discharge plot? After trying both chartConfig and dischargeChartConfig, they both worked, but I think dischargeChartConfig makes more sense if it doesn't come through, then the spinner will show from what I understand.

Reply:
I agree that it should be `dischargeChartConfig`. In the latest commit, it looked like `dischargeChartConfig` got renamed to `chartConfig`. Maybe you need to push up your latest changes to GitHub?